android/assets/jsons/translations/Dutch.properties

```diff
@@ -2608,178 +2608,178 @@ Hildesheim = Hildesheim
  # Requires translation!
-Greetings to you, exalted King Sejong the Great, servant to the people and protector of the Choson Dynasty! Your glorious vision of prosperity and overwhelming benevolence towards the common man made you the most beloved of all Korean kings. From the earliest days of your reign, the effort you took to provide a fair and just society for all was surpassed only by the technological advances spurred onwards by your unquenched thirst for knowledge. Guided by your wisdom, the scholars of the Jade Hall developed Korea's first written language, Hangul, bringing the light of literature and science to the masses after centuries of literary darkness. = 
+Greetings to you, exalted King Sejong the Great, servant to the people and protector of the Choson Dynasty! Your glorious vision of prosperity and overwhelming benevolence towards the common man made you the most beloved of all Korean kings. From the earliest days of your reign, the effort you took to provide a fair and just society for all was surpassed only by the technological advances spurred onwards by your unquenched thirst for knowledge. Guided by your wisdom, the scholars of the Jade Hall developed Korea's first written language, Hangul, bringing the light of literature and science to the masses after centuries of literary darkness. = Groeten aan jou, verheven koning Sejong de grote, dienaar van de mensen en beschermer van de Choson-dynastie! Je glorieuze visie van welvaart en overweldigende welwillendheid tegenover de gewone man maakte je de meest geliefde van alle Koreaanse koningen. Van de vroegste dagen van je regering, de moeite die je nam om een eerlijke en rechtvaardige samenleving voor iedereen te bieden, werd alleen overtroffen door de technologische vooruitgang die door je ongegetende dorst naar kennis wordt aangespoord. Geleid door je wijsheid ontwikkelden de geleerden van de Jade Hall de eerste geschreven taal van Korea, Hangul, waardoor het licht van de literatuur en de wetenschap tot de massa's na eeuwen van literaire duisternis naar de massa's brengen.
  # Requires translation!
-Honorable Sejong, once more the people look to your for guidance. Will you rise to the occasion, bringing harmony and understanding to the people? Can you once again advance your kingdom's standing to such wondrous heights? Can you build a civilization that stands the test of time? = 
+Honorable Sejong, once more the people look to your for guidance. Will you rise to the occasion, bringing harmony and understanding to the people? Can you once again advance your kingdom's standing to such wondrous heights? Can you build a civilization that stands the test of time? = Eervolle Sejong, nogmaals, de mensen kijken naar uw begeleiding. Krijgt u tot de gelegenheid, het brengen van harmonie en begrip voor de mensen? Kunt u nogmaals uw koninkrijk vooruitgaan voor zo'n wonderlijke hoogtes? Kunt u een beschaving bouwen die de test van tijd weerstaat?
```

Comment:
'Will you rise to the occasion' betekent vgm meer zoiets als 'Zal je tegen deze situatie opgewassen zijn'